sdk/ml/azure-ai-ml/azure/ai/ml/entities/__init__.py

```diff
@@ -244,7 +296,16 @@
 from ._workspace.private_endpoint import EndpointConnection, PrivateEndpoint
 from ._workspace.serverless_compute import ServerlessComputeSettings
 from ._workspace.workspace import Workspace
-from ._workspace.workspace_keys import ContainerRegistryCredential, NotebookAccessKeys, WorkspaceKeys
+from ._workspace._ai_workspaces.capability_host import (
+    CapabilityHost,
+    CapabilityHostKind,
```

Comment:
expose this through `azure.ai.ml.constants`
```suggestion

```

Reply:
this is not require. should have consistent exposure. in same file, see example of ._workspace.networking. isolationmode exposed through networking and not from constant